### ISSUE TITLE
Asymmetrical neck

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9835]
**Preferred term label:**
Asymmetrical neck

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012866

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr  I do not have access to the WHO book. Could you provide a PMID with a Figure that illustrates this finding and suggest a textual definition?

**Comment by pnrobinson** [comment]
@MagoWyr 

adding term as
HP:0034958
Label: Asymmetrical insertion of the midpiece into the head
Synonyms: Asymmetrical insertion of sperm neck, Asymmetrical insertion of spermatozoon neck
definition: Insertion of the neck and midpiece into the head of a spermatozoon on an axis that diverges from the midline axis of the spermatozoon head. This term applies if the proportion of spermatozoa with this defect is above the upper limit of normal.


(Note we cannot call the term Asymetrical neck because of the danger it would be interpreted as the neck that connects the head to the body, terms need to be unambiguous outside of the HPO hierarchy)

- Please reopen this issue if the definition requires revision
